examples/turbo-rspack-mf/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Could you please add another entry on the repository root `README.md` for this turbo project? It's important that we keep it up-to-date with new projects.

Reply:
added